java/ql/src/experimental/Security/CWE/CWE-749/UnsafeAndroidAccess.ql

```diff
@@ -0,0 +1,139 @@
+/**
+ * @name Unsafe resource loading in Android webview
+ * @description JavaScript rendered inside WebViews can access any protected application file and web resource from any origin
+ * @kind path-problem
+ * @tags security
+ *       external/cwe/cwe-749
+ *       external/cwe/cwe-079
+ */
+
+import java
+import semmle.code.java.frameworks.android.Intent
+import semmle.code.java.frameworks.android.WebView
+import semmle.code.java.dataflow.FlowSources
+
+/**
+ * Allow universal access methods in the WebSettings class
```

Comment:
Rename this, since it spots both any-local-file and universal access. Perhaps `CrossOriginAccessMethod`?

Reply:
Renamed